packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/ecs/eventhandler-image/Dockerfile

```diff
@@ -1,5 +1,3 @@
-FROM public.ecr.aws/lambda/python:3.6
-EXPOSE 8000
-WORKDIR /src
-ADD . /src
-CMD python3 index.py
+FROM --platform=x86-64 public.ecr.aws/docker/library/python:3.12
+ADD index.py .
+CMD [ "python3", "./index.py" ]
```

Comment:
I needed `--platform=x86-64` since I am building on a Mac M2.

Reply:
Instead of changing the image have you tried changing `CMD`:
```
CMD [ "index.handler" ]
```
and defining a handler function in `index.py`:
```
import os
import print

def handler(event, context):
    print('Hello from ECS!')
    pprint.pprint(dict(os.environ))
```
?

Reply:
I did try that but didn't spend a bunch of time on it. I can revisit.. but my thinking is, why is it a Lambda image when this is running on ECS and not in Lambda?

Reply:
Good point, thanks for clarifying.